elements/lisk-transactions/README.md

```diff
@@ -2,6 +2,81 @@
 
 @liskhq/lisk-transactions is containing everything related to transactions according to the Lisk protocol
 
+## Introduction
+
+Transactions are the essential part of the blockchain applications created using Lisk SDK.
+
+The Lisk SDK provides a base transaction from which developers can inherit and extend from,
+in which the application-specific business logic can be defined according to an abstract interface that is common across all transaction types.
+
+All of the default for Lisk SDK transactions are implementing the abstract interface of base transaction, therefore can be used as a role model for the custom transactions. All of them implement a different use-case of the Lisk network, i.e:
+
+0. Balance transfer (type 0),
+1. Second signature registration (type 1)
+1. Delegate registration (type 2)
+1. Delegate vote (type 3)
+1. Multisignature account registration (type 4)
+
+First 10 transaction types are reserved for the Lisk protocol, don't use them to register custom transactions.
+
+### Lifecycle
+
+The lifecycle of a transaction in general in Lisk SDK can be summarized as follows:
+
+1. A transaction is created and signed (off-chain). The script to do it is in `src/create_and_sign.ts`.
+2. The transaction is sent to a network. This can be done by a third party tool (like `curl` or `Postman`), but also using Lisk Commander, Lisk Hub or Mobile. All of the tools need to be authorized to access an HTTP API of a network node.
+3. A network node receives a transaction and after a lightweight schema, verification adds it to a transaction pool.
+4. In TransactionPool the transaction gets `validated`, also `validatedAgainstOtherTransactions` of the same type.
+5. Valid transactions go to the `prepare` step of the StateStore, which to limit the I/O database operations prepares all the information relevant to properly `validate`, `apply` or `undo` the transaction. The store with the prepared data is a parameter of the mentioned methods.
```

Comment:
4b. Firstly, transactions are validated against the transaction type. On this step, only static checks are performed. These include schema validation and signature validation.